.github/workflows/phpunit-tests.yml

```diff
@@ -114,37 +114,38 @@ jobs:
           node-version-file: '.nvmrc'
           cache: npm
 
-      - name: Install Dependencies
+      - name: Install npm dependencies
         run: npm ci
 
-      # This date is used to ensure that the Composer cache is refreshed at least once every week.
-      # http://man7.org/linux/man-pages/man1/date.1.html
-      - name: "Get last Monday's date"
-        id: get-date
-        run: echo "::set-output name=date::$(/bin/date -u --date='last Mon' "+%F")"
-
-      - name: Get Composer cache directory
-        id: composer-cache
-        run: echo "::set-output name=dir::$(composer config cache-files-dir)"
-
-      - name: Cache Composer dependencies
-        uses: actions/cache@9b0c1fce7a93df8e3bb8926b0d6e9d89e92f20a7 # v3.0.11
-        env:
-          cache-name: cache-composer-dependencies
+      ##
+      # This allows Composer dependencies to be installed using a single step.
+      #
+      # Since the tests are currently run within the Docker containers where the PHP version varies,
+      # the same PHP version needs to be configured for the action runner machine so that the correct
+      # dependency versions are installed and cached.
+      ##
+      - name: Setup PHP
+        uses: shivammathur/setup-php@e04e1d97f0c0481c6e1ba40f8a538454fe5d7709 # v2.21.2
         with:
-          path: ${{ steps.composer-cache.outputs.dir }}
-          key: ${{ runner.os }}-php-${{ matrix.php }}-date-${{ steps.get-date.outputs.date }}-composer-${{ hashFiles('**/composer.json') }}
+          php-version: '${{ matrix.php }}'
+          coverage: none
 
       - name: Install Composer dependencies
-        run: |
-          docker-compose run --rm php composer --version
+        uses: ramsey/composer-install@f680dac46551dffb2234a240d65ae806c2999dd6 # v2.1.0
+        with:
+          dependency-versions: 'highest'
+          composer-options: "${{ matrix.php == '8.2' && '--ignore-platform-req=php+' || '' }}"
 
-          # Install using `composer update` as there is no `composer.lock` file.
-          if [ ${{ env.LOCAL_PHP }} == '8.2-fpm' ]; then
-            docker-compose run --rm php composer update --ignore-platform-req=php+
-          else
-            docker-compose run --rm php composer update
-          fi
+      - name: General debug information
+        run: |
+          npm --version
+          node --version
```

Comment:
The latest version of `actions/setup-node` - `3.5.1`, which came out today, will automatically show this information already.

See: https://github.com/actions/setup-node/releases/tag/v3.5.1

Reply:
Makes sense to me.